docs/en_US/Release.md

```diff
@@ -95,18 +96,18 @@
 
 ### Major Features
 
-* [Version checking](https://github.com/Microsoft/nni/blob/master/docs/en_US/PaiMode.md#version-check)
+* [Version checking](TrainingService/PaiMode.md)
```

Comment:
please make sure that this link still works at https://github.com/microsoft/nni/releases